core/base/explicitTriangulation/ExplicitTriangulation.cpp

```diff
@@ -864,6 +864,40 @@ int ExplicitTriangulation::preconditionDistributedTriangles() {
   return 0;
 }
 
+int ExplicitTriangulation::preconditionDistributedVertices() {
+  if(this->hasPreconditionedDistributedVertices_) {
+    return 0;
+  }
+  if(!ttk::isRunningWithMPI()) {
+    return -1;
+  }
+  if(this->cellGid_ == nullptr) {
+    this->printWrn("Missing global identifiers on cells");
+    return -2;
+  }
+
+  if(this->getDimensionality() != 2 && this->getDimensionality() != 3) {
```

Comment:
dimension 1 should be supported in principle.
(I know some ttk filters are buggy in 1D)

Reply:
it shouldn't really matter of what dimension the dataset is when only looking at the ids, so different dimensions shouldn't make any problems